app/models/stats/claim_stats.rb

```diff
@@ -0,0 +1,11 @@
+module Stats
+  class ClaimStats < Claim
+    scope :started, -> { where.not(password_digest: nil, state: 'submitted').within_timeframe }
+    scope :started_count, -> { started.count }
+
+    scope :completed, -> { where(state: 'submitted').within_timeframe }
+    scope :completed_count, -> { completed.count }
+
+    scope :within_timeframe, -> { where('created_at > ?', 92.days.ago) }
```

Comment:
I think rather than assign a constant to the number of days here I'll just change the scope name. Apart from that as this data is required for a single user who know the time range already I'm not going to change the json output to give any indication of date ranges. I think if we need to offer data to more 'things' then we should readdress this then and think about the API a little more. This is also the reason why I bundled the two values in a single payload.